<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Set the Filtered data's attributes:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>281</width>
          <height>179</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="1">
             <widget class="QListWidget" name="listWidget_Items">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>GET_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GET_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PUT_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PUT_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CREATE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CREATE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CLOSE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CLOSE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>QSIZE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>QSIZE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MAXQSIZE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MAXQSIZE_END</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Items:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Example:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QListWidget" name="listWidget_Example">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>GET_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GET_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PUT_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>PUT_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CREATE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CREATE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CLOSE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CLOSE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>QSIZE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>QSIZE_END</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MAXQSIZE_START</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MAXQSIZE_END</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>179</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>351</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>43</y>
           <width>35</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Font:</string>
         </property>
        </widget>
        <widget class="QFontComboBox" name="fontComboBox_Font">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>43</y>
           <width>182</width>
           <height>27</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>47</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Action:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_Action">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>10</y>
           <width>101</width>
           <height>27</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>GET_START</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GET_END</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PUT_START</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PUT_END</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CREATE_START</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CREATE_END</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CLOSE_START</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CLOSE_END</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QSIZE_START</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QSIZE_END</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MAXQSIZE_START</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MAXQSIZE_END</string>
          </property>
         </item>
        </widget>
        <widget class="QPushButton" name="pushButton_TextColour">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>76</y>
           <width>100</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Set...</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>109</y>
           <width>129</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>background colour:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_BackgroundColour">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>109</y>
           <width>100</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Set...</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>76</y>
           <width>78</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Text colour:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_Reset">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>143</y>
           <width>100</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>331</width>
           <height>16</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_Settings">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>260</y>
           <width>331</width>
           <height>63</height>
          </rect>
         </property>
         <property name="title">
          <string>Settings:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_Export">
            <property name="text">
             <string>Export...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Import">
            <property name="text">
             <string>Import...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton_LoadFrom">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>212</y>
           <width>100</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Load from...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_SaveAs">
         <property name="geometry">
          <rect>
           <x>145</x>
           <y>177</y>
           <width>100</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Save as...</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_TextColourForeground">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>76</y>
           <width>27</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_TextColourBackground">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>109</y>
           <width>27</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
     <zorder>frame</zorder>
     <zorder>frame_3</zorder>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_Reject">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_Accept">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
